src/components/Button/Button.stories.tsx

```diff
@@ -7,8 +7,8 @@ const args = {
   onClick: () => {
     console.log("clicked");
   },
-  preIcon: <FiArrowUpRight />,
-  posIcon: <FiArrowUpRight />,
+  preIcon: <FiArrowUpRight size={24} />,
+  posIcon: <FiArrowUpRight size={24} />,
```

Comment:
We can't allow user to change this icons size. Otherwise we are breaking Luca's design. The icon size is fixed directly under the components icon.